website/content/en/docs/faq/_index.md

```diff
@@ -230,6 +231,25 @@ mounts:
 
 Run `limactl edit <INSTANCE>` to open the YAML editor for an existing instance.
 
+#### "Filesystem is unmounted after upgrading Lima to v1.0 (pre-release)"
+
+Lima v1.0 (pre-release) changed the default mount type for QEMU from `reverse-sshfs` to `9p`.
+
+The `9p` mount type is known to be incompatible with the following guest operating systems:
+- AlmaLinux, CentOS Stream, Oracle Linux, and RockyLinux
+- Debian GNU/Linux
+- openSUSE
+
+A new instance of these OS still use `reverse-sshfs` by default.
+However, an existing instance created with a previous version of Lima may potentially need
+running the following command (usually not needed):
+
+```
+limactl edit --mount-type=reverse-sshfs <NAME>
+```
```

Comment:
When will this be needed? How could the instances have started if the mount type was not supported?

Reply:
Practically never.
Just added a hint for potential bugs.